statistics/histogram.go

```diff
@@ -222,7 +230,17 @@ func (hg *Histogram) updateLastBucket(upper *types.Datum, count, repeat int64) {
 	len := hg.Len()
 	hg.Bounds.TruncateTo(2*len - 1)
 	hg.Bounds.AppendDatum(0, upper)
-	hg.Buckets[len-1] = Bucket{Count: count, Repeat: repeat}
+	hg.Buckets[len-1].Count = count
+	hg.Buckets[len-1].Repeat = repeat
+}
+
+func (hg *Histogram) updateLastBucketV2(upper *types.Datum, count, repeat int64) {
```

Comment:
How about merging this method with `updateLastBucket` to `updateLastBucket(upper, count, repeat, useBucektNDV)`?